Cargo.lock

```diff
@@ -344,15 +382,37 @@ checksum = "71655c45cb9845d3270c9d6df84ebe72b4dad3c2ba3f7023ad47c144e4e473a5"
 dependencies = [
  "atty",
  "bitflags 1.3.2",
- "clap_derive",
- "clap_lex",
- "indexmap",
+ "clap_derive 3.2.18",
+ "clap_lex 0.2.4",
+ "indexmap 1.9.3",
  "once_cell",
  "strsim",
  "termcolor",
  "textwrap",
 ]
 
+[[package]]
+name = "clap"
+version = "4.4.18"
```

Comment:
Huh, I'm guessing humility itself is still on clap 3? (This is not me asking you to change that -- iirc porting from 3 to 4 is nontrivial)

Reply:
yeah, i believe the `cargo-readme` binary depends on `clap` v4, thus the duplicate dep. i'm happy to look into a potential `clap` v4 update separately, if you'd like!